src/main/java/nl/fuchsia/services/ZaakService.java

```diff
@@ -64,4 +66,15 @@ public Zaak addZaak(ZaakAddDto zaakAddDto) {
 	public Zaak getZaakById(Integer zaakNr) {
 		return zaakRepository.getZaakById(zaakNr);
 	}
-}
+
+	public List<Zaak>  getZakenByPersoon(Integer persoonnr) {
+
+		Persoon persoon = persoonRepository.getPersoonById(persoonnr);
+
+		if (persoon == null) {
+			throw new NullException("Persoonnr " + persoonnr + " bestaat niet");
```

Comment:
Ik zou deze exceptie denk ik hernoemen naar 'NotFoundException' o.i.d., ik vind de naam nu wat onduidelijk. Het zegt wel dat het resultaat null is, maar niet _waarom_.

Reply:
Aangepast